client/src/store/modules/projects/getters.ts

```diff
@@ -0,0 +1,22 @@
+import {GET_ADDONS} from './project-types';
+import {IProjectState} from './index';
+import {Types} from './constant-types';
+
+export const getters = {
+
+  /**
+   * Get properties only for
+   * project filter
+   * @param {IProjectState} state
+   * @returns {{}}
+   */
+  [GET_ADDONS](state: IProjectState) {
+    return {
```

Comment:
Not sure if this getter is right - usually getters are functions like:
[Types.PRODUCTION_LINE]: state => state.lines

Reply:
Idea behind, Types.PRODUCTION_LINE - headers of accordion, lines/ domains ets are checkboxes values, getter in order to iterate among fewer number of keys (here we have no projects and loading)